autoload/lsp/omni.vim

```diff
@@ -332,141 +295,41 @@ function! lsp#omni#get_vim_completion_item(...) abort
-    try
-        let l:user_data = json_decode(v:completed_item['user_data'])
-    catch
-        " do nothing if user_data is not json type string.
-        doautocmd User lsp_complete_done
-        return
-    endtry
-
-    if type(l:user_data) != type({})
-        doautocmd User lsp_complete_done
-        return
-    endif
-
-    let l:all_text_edits = []
-
-    " if newText contains snippet markers, remove all them.
-    let l:snippet_marker_pos = -1
-
-    " expand textEdit range, for omni complet inserted text.
-    let l:text_edit = get(l:user_data, s:user_data_key, {})
-    if !empty(l:text_edit)
-        let l:expanded_text_edit = s:expand_range(l:text_edit, strchars(v:completed_item['word']))
-        " InsertTextFormat:Snippet
-        if get(l:user_data, s:user_data_insert_format_key, 0) == 2
-            let l:new_text = l:expanded_text_edit['newText']
-            let l:marker_pattern = '\<\$[0-9]\+\|\${[^}]\+}\>'
-            let l:snippet_marker_pos = matchstrpos(l:new_text, l:marker_pattern)[1] - 1
-            let l:expanded_text_edit['newText'] = substitute(l:new_text, l:marker_pattern, '', 'g')
-        endif
-        call add(l:all_text_edits, l:expanded_text_edit)
-    endif
-
-    if has_key(l:user_data, s:user_data_additional_edits_key)
-        let l:all_text_edits += l:user_data[s:user_data_additional_edits_key]
-    endif
-
-    " save cursor position in a mark, vim will move it appropriately when
-    " applying edits
-    let l:saved_mark = getpos("'a")
-    " move to end of newText but in two steps (as column may not exist yet)
-    let [l:pos, l:col_offset] = s:get_cursor_pos_and_edit_length(l:text_edit)
-    call setpos("'a", l:pos)
-
-    " apply textEdits
-    if !empty(l:all_text_edits)
-        call lsp#utils#text_edit#apply_text_edits(lsp#utils#get_buffer_uri(), l:all_text_edits)
-        " When user typed something character while popup menu is shwon, vim
-        " insert typed-character after CompleteDone occured. but the character
-        " should not be duplicated since the textEdit include the character.
-        " this remove the following character.
-        if l:snippet_marker_pos != -1
-            let l:oldpos = line('.')
-            let l:oldline = getline('.')
-            call timer_start(1, {_-> [
-        \    setline(l:oldpos, l:oldline),
-        \    execute('redraw', 1),
-        \    execute('doautocmd User lsp_complete_done', 1),
-        \] })
-        endif
-        return
-    endif
-
-    let l:pos = getpos("'a")
-    if l:snippet_marker_pos >= 0
-        let l:pos[2] += l:snippet_marker_pos
-    else
-        let l:pos[2] += l:col_offset
-    endif
-    call setpos("'a", l:saved_mark)
-    call setpos('.', l:pos)
-
-    doautocmd User lsp_complete_done
+"
+" Clear internal user_data map.
+"
+" This function should call at `CompleteDone` only if not empty `v:completed_item`.
+"
+function! lsp#omni#clear_managed_user_data_map() abort
```

Comment:
I started changing coding standards. Can you change this to start with underscore so that it is treated as internal for vim-lsp. `lsp#omni#_clear_managed_user_data_map()`